__tests__/classes/CanvasRenderingContext2D.reset.js

```diff
@@ -0,0 +1,24 @@
+let canvas;
+let ctx;
+
+beforeEach(() => {
+  canvas = document.createElement('canvas');
+  ctx = canvas.getContext('2d');
+  canvas.width = 400;
+  canvas.height = 300;
+});
+
+describe('reset', () => {
+  it('should be a function', () => {
+    expect(typeof ctx.reset).toBe('function');
+  });
+
+  it('should be callable', () => {
+    ctx.rect();
+    expect(ctx.reset).toHaveBeenCalled();
+  });
+
+  it('should throw if any parameters are given', () => {
+    expect(() => ctx.rect(1)).toThrow(TypeError);
```

Comment:
test the throw message.

Reply:
So, something like `expect(() => ctx.reset(1)).toThrow(/Failed to execute 'reset' on/);`?